docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,326 @@ title: React Native SDK Migration Guide
+
+To migrate to `@amplitude/analytics-react-native`, update your dependencies and instrumentation.
+
+## Terminology
+
+* `@amplitude/react-native`: Maintenance React Native SDK
+* `@amplitude/analytics-react-native`: New React Native SDK
+
+## Dependency
+
+Uninstall `@amplitude/react-native` by either `yarn remove @amplitude/react-native` or deleting your dependency list in package.json
+
+=== "@amplitude/react-native"
+
+    ```json
+    {
+      "dependencies": {
+        "@amplitude/react-native": "^2"
+      }
+    }
+    ```
+
+Follow [the installation section](../#installation) to add new React Native SDK dependencies.
+
+## Instrumentation
+
+New React Native SDK offers an API to instrument events. To migrate to it, you need to update a few calls. The following sections detail which calls have changed.
+
+### Initialization
+
+`getInstance()` in the maintenance SDK only accepts API key and has been removed. To initialize the SDK, call `init()`, with the user ID and configuration parameters.
+
+=== "@amplitude/react-native"
+
+    ```typescript
+    import { Amplitude } from '@amplitude/react-native';
+
+    Amplitude.getInstance().init(API_KEY)
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { init } from '@amplitude/analytics-react-native';
+
+    init(API_KEY, OPTIONAL_USER_ID, config);
+    ```
+
+### Configuration
+
+Maintenance React Native SDK runs on top of the maintenance Android SDK and maintenance iOS SDK and uses settable config options. However new React Native SDK instance has an configuration object in it so that we can set configurations directly on initialization via the instance.
+
+|@amplitude/react-native|@amplitude/analytics-react-native|
+|-|-|
+| `enableCoppaControl()` | NOT SUPPORTED |
+| `disableCoppaControl()` | NOT SUPPORTED |
+| `setAdvertisingIdForDeviceId()` | NOT SUPPORTED |
+| `setAppSetIdForDeviceId()` | NOT SUPPORTED |
+| `setOptOut()` | both `setOptOut()` and `config.optOut` are supported |
+| `trackingSessionEvents()` | `config.trackingSessionEvents` |
+| `setUseDynamicConfig()` | NOT SUPPORTED |
+| `setMinTimeBetweenSessionsMillis()`| similar to `config.sessionTimeout`, but they are not completely the same. Learn more about [session management](./#session-management)|
+| `setServerZone()` | `config.serverZone` |
+| `setServerUrl()` | `config.serverUrl` |
+| `setEventUploadMaxBatchSize()` | `config.flushQueueSize` |
+| `setEventUploadPeriodMillis()` | `config.flushIntervalMillis` |
+| `setEventUploadThreshold()` | `config.flushQueueSize` |
+| `enableLogging()`| Logging is enabled and cannot be turned off. However, you can set `config.logLevel` and customize `config.loggerProvider` |
+| `setLogLevel()`| `config.logLevel` |
+| `addLogCallback()` | You can add callback by customizing `config.loggerProvider` |
```

Comment:
`addLogCallback` is designed to log error on android native side. I feel loggerProvider might not be able to make support it in the same way.

Reply:
Change it to "It's not fully supported but you can customize a logger by setting `config.loggerProvider`."